src/components/Interpretations/InterpretationModal/InterpretationModal.js

```diff
@@ -22,16 +22,18 @@ import { useModalContentWidth } from './useModalContentWidth.js'
 
 const modalCSS = css.resolve`
     aside {
-        min-width: 600px;
-        min-height: 600px;
-        max-width: 90vw !important;
-        max-height: 90vh !important;
+        max-width: calc(100vw - 128px) !important;
+        max-height: calc(100vh - 128px) !important;
         width: auto !important;
         height: auto !important;
+        overflow-y: hidden;
     }
     aside.hidden {
         display: none;
     }
+    aside > :global(div) > :global(div) {
+        max-height: 100%;
+    }
```

Comment:
This part can be removed when https://github.com/dhis2/ui/pull/1013 is merged and we upgrade @dhis2/ui.

Reply:
This was done now 👍